src/scenekit.cs

```diff
@@ -121,6 +121,11 @@ interface SCNAnimatable {
 		[Mac (10,10)]
 		[Export ("removeAnimationForKey:fadeOutDuration:")]
 		void RemoveAnimation (NSString key, nfloat duration);
+
+		[TV (10, 0), Mac (10, 12), iOS (10, 0)]
+		[Abstract]
```

Comment:
@spouliot  Yep agree and spotted some other cases in other protocols but this one in particular has some more context let me paste more of it

```
#if XAMCORE_2_0
		[Abstract]
#endif
		[Mac (10,9)]
		[Export ("isAnimationForKeyPaused:")]
		bool IsAnimationPaused (NSString key);

#if XAMCORE_2_0
		[Abstract]
#endif
		[Mac (10,10)]
		[Export ("removeAnimationForKey:fadeOutDuration:")]
		void RemoveAnimation (NSString key, nfloat duration);

		[TV (10, 0), Mac (10, 12), iOS (10, 0)]
		[Abstract]
		[Export ("setSpeed:forAnimationKey:")]
		void SetSpeed (nfloat speed, NSString key);
```

All of the members of this protocol in `XAMCORE_2_0` (IOW Unified) are decorated with `Abstract` and I just followed suit with the exception of adding a `XAMCORE_2_0` block because IIRC `classic` will be no more once this hits stable. Let me know if you want me change it anyways.